src/main/scala/it/unibo/authsim/library/user/builder/util/Util.scala

```diff
@@ -0,0 +1,16 @@
+package it.unibo.authsim.library.user.builder.util
+
+import scala.util.Random
+
+/**
+ * Object for usefull functions
+ */
+object Util {
```

Comment:
I don't like the name `Util` as it does not provide additional useful information (if the code is present, it better be useful).

Maybe a better name could be `RandomStringGenerator` (just for the file and the object, the package is ok).